jobs/delorean/jenkinsfiles/2.0/ews/discovery/umb/Jenkinsfile

```diff
@@ -55,15 +55,26 @@ pipeline {
 
         stage('Process Manifest Images') {
             steps {
-                //ToDo Process the manifests CSV to update release image registries for any non GA images (Generates image_mirror_mapping file)
-                echo "Process Manifest Images"
+                dir("${env.WORKSPACE}"){
+                    sh "scripts/process-image-manifests registry-proxy.engineering.redhat.com/rh-osbs/3scale-amp2-3scale-rhel7-operator-metadata:1.11.0-3 3scale 3scale-operator"
```

Comment:
Need to have these variables passed.

Reply:
Where are these going to come from? Did we not set these as part of the work to  "Process UMB message"?

Reply:
We need to commit changes after each stage that wil be making modifications, we had a library function to do this previously https://github.com/integr8ly/ci-cd/blob/master/jobs/delorean/jenkinsfiles/1.0/ews/discovery/ga/github/Jenkinsfile#L248

```
    gitCommitWhenChanges("Updated ${product} product version") { msgs ->
        sh "scripts/process-image-manifests $image $product $product-operator"
    }
```

Reply:
👍 Just pushed picking up the image. Will put in the gitCommit stuff too.